haystack/components/embedders/sentence_transformers_document_embedder.py

```diff
@@ -95,6 +96,11 @@ def __init__(  # noqa: PLR0913
         :param tokenizer_kwargs:
             Additional keyword arguments for `AutoTokenizer.from_pretrained` when loading the tokenizer.
             Refer to specific model documentation for available kwargs.
+        :param precision:
+            The precision to use for the embeddings. Can be “float32”, “int8”, “uint8”, “binary”, or “ubinary”.
```

Comment:
```suggestion
            The precision to use for the embeddings.
```

Possible values can be inferred from the signature and we prefer to avoid including them in docstrings.